src/compiler/output-targets/dist-custom-elements-bundle/custom-elements-build-conditionals.ts

```diff
@@ -1,3 +1,4 @@
+// TODO(STENCIL-561): fully delete dist-custom-elements-bundle code
```

Comment:
Nit/fun fact - this file is also used in `src/compiler/output-targets/dist-custom-elements/index.ts` - can we update the TODO to move it to `dist-custom-elements/` and rename it?

Reply:
that is indeed "fun"